olympus.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/sh
 # Olympus launch script bundled with Linux and macOS builds.
 
-cd "$(dirname "$0")"
+cd "$(dirname "$0")" || exit 1
```

Comment:
Can `cd` even fail in this case when the directory theoretically must exist to reach this point to begin with?